packages/react-native/src/notifier.js

```diff
@@ -1,6 +1,14 @@
 const NativeModules = require('react-native').NativeModules
 const NativeClient = NativeModules.BugsnagReactNative
 
+const REMOTE_DEBUGGING_WARNING = `Bugsnag cannot initialise synchronously when running in the remote debugger.
```

Comment:
Message looks good to me. Just a reminder about initialise -> initialize as you mentioned.